### DISCUSSION TITLE
执行编译命令taro build —-type harmony —-watch，提示Error: 不存在编译平台 harmony

### DISCUSSION THREAD

**Discussion by lihao29** [NervJS/taro#13608] [鸿蒙 && OpenHarmony 适配小组]
按照官方文档执行：
https://taro-docs.jd.com/docs/harmony#%E4%BD%BF%E7%94%A8-taro-%E5%BC%80%E5%8F%91%E9%B8%BF%E8%92%99-js-ui
到第5步编译运行命令taro build —-type harmony —-watch，提示Error: 不存在编译平台 harmony
![image](https://user-images.githubusercontent.com/130196996/230699033-43538481-fa7d-4f16-bb4d-2b3b7b8ea698.png)
@tarojs/cli版本相关信息如下：
![image](https://user-images.githubusercontent.com/130196996/230699081-31932b36-fb18-425d-984d-4e478992ab23.png)

**Comment by TheKonka** [comment]
![image](https://user-images.githubusercontent.com/22173084/230705205-32dee263-808a-46b8-a518-418f90851113.png)
package.json 里面加上命令然后用 npm 运行看看

**Reply by lihao29** [reply]
试了下还是不行，您那边可以的吗？
![image](https://user-images.githubusercontent.com/130196996/230713301-70afdc08-e935-4cef-b9e4-94edd1fe11b0.png)
![image](https://user-images.githubusercontent.com/130196996/230713338-1c8f2647-a66d-4fa6-a307-d56cf576302c.png)
![image](https://user-images.githubusercontent.com/130196996/230713379-f5fb8f0c-3f3c-4a66-a837-15dab49188ba.png)

**Comment by TheKonka** [comment]
![image](https://user-images.githubusercontent.com/22173084/230758525-8690a423-0b49-466a-8154-5f3c4db6f57b.png)
是，没遇到你那个报错，可能与系统环境有关

**Comment by huajian123** [comment]
这个问题有解决吗
